<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HexWalkMain</class>
 <widget class="QMainWindow" name="HexWalkMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>786</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>HexWalk</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QHexEdit" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>160</height>
       </size>
      </property>
      <property name="title">
       <string>Info</string>
      </property>
      <widget class="QLabel" name="hexlbl">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>90</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Hex:</string>
       </property>
      </widget>
      <widget class="QLabel" name="declbl">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>60</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>IntBE:</string>
       </property>
      </widget>
      <widget class="QLabel" name="intlelbl">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>90</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>IntLE:</string>
       </property>
      </widget>
      <widget class="QLabel" name="binlbl">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>120</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Bin:</string>
       </property>
      </widget>
      <widget class="QLabel" name="hexTextedit">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>90</y>
         <width>281</width>
         <height>31</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
      <widget class="QLabel" name="decTextedit">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>60</y>
         <width>311</width>
         <height>31</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
      <widget class="QLabel" name="intleTextedit">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>90</y>
         <width>311</width>
         <height>31</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
      <widget class="QLabel" name="binTextedit">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>120</y>
         <width>711</width>
         <height>31</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
      <widget class="QLabel" name="floatlbl_le">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>30</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>FloatLE:</string>
       </property>
      </widget>
      <widget class="QLabel" name="floatTextedit_le">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>30</y>
         <width>281</width>
         <height>31</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
      <widget class="QLabel" name="asciilbl">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>30</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>ASCII:</string>
       </property>
      </widget>
      <widget class="QLabel" name="asciiTextEdit">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>30</y>
         <width>251</width>
         <height>31</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
      <widget class="QLabel" name="sellbl">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Selected bytes:</string>
       </property>
      </widget>
      <widget class="QLabel" name="selTextedit">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>30</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QCheckBox" name="signedcb">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>88</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Signed</string>
       </property>
      </widget>
      <widget class="QLabel" name="floatTextedit_be">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>60</y>
         <width>281</width>
         <height>31</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
      <widget class="QLabel" name="floatlbl_be">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>60</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>FloatBE:</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QHexEdit</class>
   <extends>QWidget</extends>
   <header>../src/qhexedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
